ggshield/cmd/common_options.py

```diff
@@ -114,7 +114,7 @@ def debug_callback(
     "--check-for-updates/--no-check-for-updates",
     is_flag=True,
     default=None,
-    help="Check for ggshield updates.",
+    help="Make a daily check for ggshield updates before executing commands.",
```

Comment:
I suggest:

```suggestion
    help="After executing commands, check if a new version of ggshield is available.",
```

I changed "before" to "after" since the check is now done just before exiting.

Reply:
Oops, I forgot about that. Nice catch !